docs/section12/Rule12-8.md

```diff
@@ -0,0 +1,40 @@
+
+# Receptacle - Rule 12-8
+
+**Rule ID:** 12-8  
+**Rule Description:** Receptacle control credit modeled correctly in User RMR and Baseline RMR are the same  
+**Rule Assertion:** User RMR = expected value, Baseline RMR = User RMR  
+**Appendix G Section:** Receptacle  
+**Appendix G Section Reference:** None  
+
+**Applicability:** All required data elements exist for B_RMR and U_RMR  
+**Applicability Checks:**  
+
+  1. Rule 12-4 = True  
+
+**Manual Check:** None  
+**Evaluation Context:** Each Data Element  
+**Data Lookup:** None  
+
+## Rule Logic:  
+
+- **Applicability Check 1:** ```if (rule-12-4.status == True):```
+- Get the receptacle load of each space in the building segment in the User model: ```For building_segment in U_RMR.building.building_segments:```  
+  - Get thermal_block from building segment: ```thermal_block in building_segment.thermal_blocks:```
+  - Get thermal_zone from thermal block: ```thermal_zone in thermal_block.zones:```
+  - Get space from thermal zone: ```space in thermal_zone.spaces:```  
+    - Get the name of the space: ```space_name = space.name```
+    - Get the receptacle load in the space: ```space_receptacle = space.equipment_internal_gains```
+      - Get the receptacle load control credit: ```receptacle_control = space_receptacle.receptacle_control_credit```
+      - Save the receptacle load schedule for all spaces in the User model: ```space_receptacle_schedules_user[space_name] = ELFH(receptacle_sch) / ( 1 - receptacle_control )```
```

Comment:
shouldn't it just be multiplied by 90%, the RPC credit is just 10%.

Also the credit only applies where it is not required by Section 8.4.2 so need to check for private offices, etc..

Reply:
Section 8.4.2 requires automatic receptacle control for 

> a. At least 50% of all 125 V, 15 and 20 amp receptacles in all private offices, conference
rooms, rooms used primarily for printing and/or copying functions, break rooms,
classrooms, and individual workstations.
> b. At least 25% of branch circuit feeders installed for modular furniture not shown on
the construction documents.

With an exception of 

> 1. Receptacles specifically designated for equipment requiring continuous operation (24/day,
365 days/year).
> 2. Spaces where an automatic control would endanger the safety or security of the room or
building occupants.

Not sure how we can check 50% and the exceptions. Do we want to just check MiscellaneousEquipment.has_automatic_control for space types other than private office, conference rooms, copy rooms, break rooms, class rooms and open office?

Reply:
Yes, I think this means that it is only for spaces other than what is listed in 8.4.2(a)

Reply:
@xingcx please make a new pull request for rule 12-8 and add a link to this discussion.